.dockerignore

```diff
@@ -0,0 +1,3 @@
+node_modules
```

Comment:
This just saves space in the docker container as Docker does not care about these files/directories.